src/writer/serve.py

```
@@ -91,6 +91,55 @@ class WriterFastAPI(FastAPI, WriterAsgi):  # type: ignore
+    return False
+
+_load_secure_module()
+
+if not SECURE_TOKENS_ENABLED:
+    class _SecureTokenManagerStub:
+        @staticmethod
+        def store_token(session_id: str, token: str) -> None:
+            pass
+        
+        @staticmethod
+        def get_token(session_id: str) -> str:
```

Ruff (0.14.3)

Unused static method argument: `session_id`

(ARG004)